api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -89,4 +89,174 @@ enum Modulation {
 
   // Binary Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying#Binary_phase-shift_keying_(BPSK)
   BPSK = 2;
+
+  // Multiple Frequency Shift Keying. https://en.wikipedia.org/wiki/Multiple_frequency-shift_keying
+  MFSK = 3;
+
+  // Quadrature Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying#Quadrature_phase-shift_keying_(QPSK)
+  QPSK = 4;
+
+  // 8-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK8 = 5;
+
+  // 16-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK16 = 6;
+
+  // 32-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK32 = 7;
+
+  // 64-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK64 = 8;
+
+  // 128-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK128 = 9;
+
+  // 256-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK256 = 10;
+
+
+  // Differential Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying#Differential_phase-shift_keying_(DPSK)
+  DPSK = 11;
+
+  // Optical Differential Phase Shift Keying. https://www.bell-labs.com/our-research/publications/203155/
```

Comment:
I was thinking "Offset DPSK" here, but I can't find it anywhere now, so let's drop it.